proto/comdex/lend/v1beta1/lend.proto

```diff
@@ -523,4 +523,51 @@ message Lend_rewards_tracker{
     (gogoproto.nullable) = false,
     (gogoproto.moretags) = "yaml:\"interest_accumulated\""
   ];
-}
\ No newline at end of file
+}
+
+message Asset_ranking{
```

Comment:
don't need this, its much better to do this calculation off chain.

Reply:
changes reverted.